scripts/globals/mobskills/Bubble_Shower.lua

```diff
@@ -9,7 +9,25 @@ require("scripts/globals/monstertpmoves");
 
 ---------------------------------------------------
 
-function onMobSkillCheck(target,mob,skill)
+function onMobSkillCheck(target,mob,skill,zone)
+	mob = mob:getID();
+	if (mob == 17129519) then
+	local mobids = {17129520,   -- King Arthro's Knights

```

Comment:
Hey thanks ill try to remember that for future projects for sure :)

On Fri, Dec 18, 2015 at 3:32 PM, TeoTwawki <notifications@github.com> wrote:

> In scripts/globals/mobskills/Bubble_Shower.lua
> <https://github.com/DarkstarProject/darkstar/pull/2493#discussion_r48081524>
> :
>
> > @@ -9,7 +9,25 @@ require("scripts/globals/monstertpmoves");
> >
> >  ---------------------------------------------------
> >
> > -function onMobSkillCheck(target,mob,skill)
> > +function onMobSkillCheck(target,mob,skill,zone)
> > +	mob = mob:getID();
> > +	if (mob == 17129519) then
> > +	local mobids = {17129520,   -- King Arthro's Knights
>
> same formatting probs I pointed out in your other PR, just so you know.
> And you need to rebase this off newer DSP, you are missing the onMobDeath
> script changes. killer: is replaced by ally: in most cases
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/2493/files#r48081524>.
>